### PR TITLE
53 site chrome

### PR DESCRIPTION
**Pull request by cdmo**
for #53 

do after #213 

Screenshot_2020-07-26 My Account Pennsylvania State University Libraries

### CODE REVIEW

**Comment by cdmo** [comment]
Last commit corrects this problem:

Screen Shot 2020-07-27 at 11 28 53 AM

**Comment by banukutlu** [comment]
@cdmo I think this needs a rebase since now #213 is merged :)

**Comment by cdmo** [comment]
Rebased 👍

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+  end
+
+  def current_year
+    Date.today.year
+  end
+
+  private
+
+    def active_controller?(name)
+      name == controller_name
```
Comment (banukutlu) [inline app/helpers/application_helper.rb:24]: 👌

**Pull request merged**